templates/libraries/jersey3/api_summary_overload.mustache

```diff
@@ -12,7 +12,8 @@
 {{/returnType}}
     * @throws ApiException if fails to make API call
 {{#isDeprecated}}
-    * @deprecated
+    * @deprecated {{#vendorExtensions.x-deprecatedInVersion}}since {{#appName}}{{{.}}}{{/appName}} v{{#vendorExtensions.x-deprecatedInVersion}}{{.}}{{/vendorExtensions.x-deprecatedInVersion}}
+    * {{#vendorExtensions.x-deprecatedMessage}}{{{.}}}{{/vendorExtensions.x-deprecatedMessage}}{{/vendorExtensions.x-deprecatedInVersion}}
```

Comment:
Is the closing tag `{{/vendorExtensions.x-deprecatedInVersion}}` correct?

If `{{#vendorExtensions.x-deprecatedMessage}}{{{.}}}{{/vendorExtensions.x-deprecatedMessage}}` is empty, there's an additional `* ` empty line here

Reply:
Correct, by design. In case this happens a new line is not noticeable IMO

Reply:
I have nevertheless updated the templates to check independently for the 2 extensions. Thank you